source/software/matlab_parallel_computing.rst

```diff
@@ -7,119 +7,99 @@ General
+    $ # Within Matlab
+    $ >> configCluster;
 
-::
+The `configCluster` function will request you to specify a cluster (`wice` or `genius`). Please use the one you are on already.
 
-   $ matlab -nodisplay
+There is a range of additional properties you can set for the cluster profile. Before you can edit these, you first need to get a handle on the cluster
+profile:
 
 ::
 
-   >> rehash toolbox;
-   >> configCluster;
-   
-You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. You have now created a cluster
-profile.
-
-Configuring additional properties
-*********************************
+    $ c=parcluster;
 
-It is possible to set some user-specific properties and save them in your cluster profile. You can always adapt these and save the profile again, without performing
-the previous step again. All of these properties are available in the ``AdditionalProperties`` property of your cluster profile. 
-   
-Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+You can then view and edit the additional properties:
 
 ::
 
-   >> c = parcluster;
-   
-``c = parcluster`` will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
-the correct profile will be chosen when using this command.
+    $ # Access
+    $ c.AdditionalProperties
+    $
+    $ #edit
+    $ c.AdditionalProperties.<property_name> = '<property_value>'
 
-If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+One of these properties you will always have to set, namely `AccountName`. Set this to the name of your credit account.
+
+Next to this, it is also possible to change the more standard parameters of the cluster profile. One of these is the `JobStorageLocation` parameter. This defaults
+to a location in your `$VSC_HOME`. It is however strongly recommended to change this to another location, e.g. `$VSC_DATA`. Changing this, or any of
+the other cluster profile parameters can be done as follows:
 
 ::
 
-   >> c = parcluster('local');
+    $ # Access
+    $ c.JobStorageLocation = '</path/to/your/custom/jobdir>'
 
-You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
-other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+Both after changing any of the additional properties or the standard parameters, you will have to save that profile to keep the changes after closing your
+session:
 
 ::
 
-   >> % View the additional properties
-   >> c.AdditionalProperties
-   >> % set the required properties
-   >> c.AdditionalProperties.AccountName = <account-name>;
-   >> c.AdditionalProperties.WallTime = '05:00:00';
```

Comment:
Specifying `WallTime` is also an interesting argument. Also, if it is possible to specify the number of nodes and cores per node and memory, if the profile allows?